### ISSUE TITLE
Add milestones to github

### ISSUE THREAD

**Issue by benwbrum** [FreeUKGen/FreeCENMigration#35]

**Comment by PatReynolds** [comment]
It seems only the creator of a repository can add milestones (or I have followed the wrong instructions - https://help.github.com/articles/creating-and-editing-milestones-for-issues-and-pull-requests/ when I get to clicking 'repositories' it tells me I don't have any.

**Comment by sean-wood** [comment]
Ben to change repository owner

**Comment by PatReynolds** [comment]
Thanks Ben.

I will try and load Github - possibly tomorrow as wifi is struggling to
load freereg2 at the moment....

But wifi is working!  We have a huge arial on the roof of our guest house
(holiday rental) and am now wired into that.  But sharing a tiny satellite
bandwidth to Cape Town makes everything slow.  I am learning lots about how
to get things to be smaller, and not to interfere with the vital process of
gmail.

Best wishes,

Pat

On Sunday, 13 September 2015, sean-wood <notifications@github.com> wrote:

> Assigned #35 <https://github.com/FreeUKGen/FreeCENMigration/issues/35> to
> @PatReynolds <https://github.com/PatReynolds>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeUKGen/FreeCENMigration/issues/35#event-408024936>.
>


-- 
*- -*

*Dr Pat Reynolds*
Executive Director
Free UK Genealogy  <http://www.freeukgenealogy.org.uk/>
A charity registered in England and Wales, number 1096940

​+44 ​1904 541411​ +44 7943 145387

36 Albemarle Road, York, YO23 1ER, UK

**Comment by PatReynolds** [comment]
I'm afraid it still says that I have no public repositories.

**Comment by benwbrum** [comment]
After a lot of struggling with permissions and group ownership, I gave up and added milestones to FreeCENMigration myself.  I used the same pattern we used for FreeREG2

**Comment by PatReynolds** [comment]
Many thanks, Ben.  You will see that I have now added milestones to the
issues in the Incoming and Defined & Prioritised groups.

Carried away by the ability to do this, I have also created issues for all
the aspects in the roadmap for release 1.1, and an action to do the same
for 1.2

With best wishes,

Pat

On Monday, 21 September 2015, Ben W. Brumfield <notifications@github.com>
wrote:

> After a lot of struggling with permissions and group ownership, I gave up
> and added milestones to FreeCENMigration myself. I used the same pattern we
> used for FreeREG2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeUKGen/FreeCENMigration/issues/35#issuecomment-142044716>
> .
>


-- 
*- -*

*Dr Pat Reynolds*
Executive Director
Free UK Genealogy  <http://www.freeukgenealogy.org.uk/>
A charity registered in England and Wales, number 1096940

​+44 ​1904 541411​ +44 7943 145387

36 Albemarle Road, York, YO23 1ER, UK
